.evergreen/config.yml

```diff
@@ -36,60 +36,51 @@ functions:
     # Make an evergreen exapanstion file with dynamic values
     - command: shell.exec
       params:
-        working_dir: "src"
+        shell: bash
         script: |
-           echo "" > expansion.yml
-
+           pushd src
```

Comment:
Being able to change the shell to bash (which is a new feature) makes this work, I'm pretty sure.